src/dali/transaction_resolver.py

```diff
@@ -125,6 +122,7 @@ def _get_originating_exchange(transaction: AbstractTransaction) -> str:
         return transaction.from_exchange
     raise Exception(f"Internal error: not a transaction: {transaction}")
 
+
```

Comment:
Remove extra newline.

Reply:
Two blank lines between functions is standard.  My versions:
- black 22.3.0
- isort 5.9.3

These versions should also match the pre-commit configuration, however I see blacken-docs has a dependency on black 20.8b1.

Reply:
Actually you're right: the newline was missing.